.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
 - '8'
-cache:
-  directories:
-  - node_modules
-  
\ No newline at end of file
+deploy:
+- provider: script
+  skip_cleanup: true
+  script: npx semantic-release
```

Comment:
You should specify the branch this should be run on (master). I think it will always run the deploy script when there's a push to a side branch if you don't.

Reply:
Like this: https://github.com/watson-developer-cloud/swift-sdk/blob/master/.travis.yml#L49